hatch_build.py

```diff
@@ -414,7 +414,7 @@
     'pendulum>=3.0.0,<4.0;python_version>="3.12"',
     "pluggy>=1.5.0",
     "psutil>=5.8.0",
-    "pydantic>=2.7.0",
+    "pydantic==git+https://github.com/pydantic/pydantic@main",
```

Comment:
What about `pydantic @ git+https://github.com/pydantic/pydantic.git@main`?

Reply:
i was out after our conversations - but let's try  it.